<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWindow</class>
 <widget class="QDialog" name="FormWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>577</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>577</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Formularz</string>
  </property>
  <widget class="QLineEdit" name="eFirstName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lFirstName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imię</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eLastName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ePhone">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lLastName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwisko</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPhone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr telefonu</string>
   </property>
  </widget>
  <widget class="QLabel" name="lNotes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Uwagi</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="eNotes">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPayment">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Przedpłata/Zapłacono</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPrice">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cena (zł/h)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ePrice">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="eRecipient">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lRecipient">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Przyjmujący</string>
   </property>
  </widget>
  <widget class="QLabel" name="lReceptionTime">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Czas przyjęcia</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="eReceptionTime">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bBook">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rezerwuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bCancel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bCancel</sender>
   <signal>clicked()</signal>
   <receiver>FormWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
